mono/utils/mono-os-wait-win32.c

```diff
@@ -168,8 +169,7 @@ mono_win32_wait_for_multiple_objects_ex (DWORD count, CONST HANDLE *handles, BOO
 	return result;
 }
 
-#if G_HAVE_API_SUPPORT(HAVE_CLASSIC_WINAPI_SUPPORT)
-
+#ifdef HAVE_API_SUPPORT_WIN32_SIGNAL_OBJECT_AND_WAIT
```

Comment:
Should be `if` not `ifdef` so that it can be explicitly stated as not present, instead of ambiguous omission.